bot.py

```diff
@@ -700,11 +700,215 @@ def cmd_shoutout(match: Match[str]) -> Response:
     )
 
 
+class YouTubeVideo(NamedTuple):
+    video_id: str
+    title: str
+    playlist_id: str
+    playlist_position: int
+
+    def chat_message(self) -> str:
+        return (
+            f'{self.title} - '
+            f'https://www.youtube.com/watch?v={self.video_id}&list={self.playlist_id}&index={self.playlist_position}'  # noqa: E501
```

Comment:
looks like these can be the short form `https://youtu.be/{self.video_id}?list={...}` (also seems that the index parameter isn't needed?)

Reply:
Good point. Yeah I'm not sure what the index param gives you, docs don't seem very helpful either. I'll drop the whole index/position concept

Reply:
You'll obviously need to DROP/ALTER the table now if you've run this branch locally, soz